client/src/main/java/com/paypal/selion/internal/platform/grid/AbstractTestSession.java

```diff
@@ -123,7 +123,7 @@ protected final void initTestSession(InvokedMethodInformation method) {
     /*
      * Returns true if SessionSharing is enforced by the client test class
      */
-    private boolean isSessionShared(InvokedMethodInformation invokedMethodInformation) {
+    private boolean canSessionShared(InvokedMethodInformation invokedMethodInformation) {
```

Comment:
Please keep the original method name intact here. the new name does not convey the intent.. If the concern was the method takes the same name as a variable name, please change the variable name